plugins/Aka/plugin.py

```diff
@@ -193,6 +194,15 @@ def get_aka_lock(self, channel, name):
                 return (bool(r[0]), r[1], r[2])
             else:
                 raise AkaError(_('This Aka does not exist.'))
+
+        def get_all(self, channel):
+            cursor = self.get_db(channel).cursor()
+            cursor.execute("""
+                SELECT id, name, alias, locked, locked_by, locked_at
+                FROM aliases;
+            """)
+            return cursor.fetchall()
+
     available_db.update({'sqlite3': SQLiteAkaDB})
 elif sqlalchemy:
```

Comment:
Additionally, this is dead code, since sqlite3 has been a core module since Python 2.5.

Reply:
Isn't it optionally part of python, though? Anyway, I think you're right, I want to get rid of the sqlalchemy backend too; it's just duplicate code with barely any benefit.

Reply:
Oh, indeed it is, though I've never seen any build without it, and everything I've seen in the last ~5 years always expect it to be there.